innovation_sweet_spots/utils/nihr/create_nihr.py

```diff
@@ -0,0 +1,37 @@
+"""
+innovation_sweet_spots.utils.nihr.create_nihr
+
+Module for creating NIHR data from source and saving into the S3 bucket
+"""
+import numpy as np 
+import pandas as pd 
+import altair as alt
+import sys
+from innovation_sweet_spots.utils.read_from_s3 import save_csv_to_s3
+
+COLUMNS = ['datasetid', 'recordid', 'geometry', 'record_timestamp',
+       'latitude', 'postcode', 'programme_stream', 'funder', 'longitude',
+       'project_status', 'project_title', 'award_holder_name', 'start_date',
+       'acronym', 'funding_stream', 'orcid', 'contracted_organisation', 'geo',
+       'programme_type', 'award_amount_from_dh', 'scientific_abstract',
+       'programme', 'project_id', 'involvement_type', 'ukcrc_value',
+       'end_date', 'plain_english_abstract', 'organisation_type',
+       'award_amount_m']
+
+def create_nihr():
```

Comment:
Hey, could you please indicate the type of the output, for example (not sure if this is the right type!):
```suggestion
def create_nihr() -> dict:
```